tests/test_http_parser.py

```diff
@@ -1205,8 +1205,8 @@ def test_parse_chunked_payload_chunk_extension(parser: Any) -> None:
     assert payload.is_eof()
 
 
-def _test_parse_no_length_or_te_on_post(loop, protocol, request_cls):
-    parser = request_cls(protocol, loop, readall=True)
+def test_parse_no_length_or_te_on_post(loop: Any, protocol: Any, request_cls: Any):
+    parser = request_cls(protocol, loop, 2**16)
```

Comment:
Could you use a named kwarg so it's visible what this arbitrary constant stands for?